Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -697,6 +698,161 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Multiple Collections Incremental R
     verifyDocs(collectionSpecs, docIDs, true);
 }
 
+TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Pull deltas from Collection SG", "[.SyncServerCollection]") {
+    _authHeader = TestUserAuthHeader;
+    const string idPrefix = timePrefix();
+    // one collection now now. Will use multiple collection when SG is ready.
+    constexpr size_t collectionCount = 1;
+
+    constexpr size_t kDocBufSize = 60;
+    constexpr int kNumDocs = 50, kNumProps = 50;
```

Comment:
please put comment why reduce the number from 1000

Reply:
will add comment and once I discuss with SGW, will create ticket if necessary